### ISSUE TITLE
Big play button doesn't show on iphone iOS 9.2 

### ISSUE THREAD

**Issue by smohadjer** [videojs/video.js#3012]
Videojs' big play button doesn't show on iPhones with latest iOS (9.2), but it shows on Android 5.1. How come? You can test this by opening videojs.com homepage on an iPhone.

**Comment by gkatsev** [comment]
This is because on iphones, when you play a video, you can only play it in the native player in fullscreen. Plus, you cannot fully remove the `controls` attribute of the video element which is necessary to make fully custom UIs on top of the video element. We've chosen to not enable our custom skin on iphones by default because it makes it the best user experience all around.
Hope that helps, thanks.

**Comment by smohadjer** [comment]
Is this something you decided in version 5.0? Before upgrading to 5 we had custom play button in our videojs even on iPhone.

**Comment by gkatsev** [comment]
I believe we had this in 4.x. Also, you could try setting `nativeControlsForTouch` to false for the `html5` tech if you would like to keep using it.
```js
var player = videojs('vid', {
  html5: {
    nativeControlsForTouch: false
  }
});
```

**Comment by smohadjer** [comment]
Your suggestion doesn't work! Open below fiddle on an iPhone and you see that it still displays the native iPhone play button over the video poster.

https://jsfiddle.net/smohadjer/89v532nz/4/embedded/result/#Result

**Comment by smohadjer** [comment]
This problem is resolved in videojs 5.12.6.
